### PR TITLE
Feature fe rooms

### PR DESCRIPTION
**Pull request by jimmy1993k**

### CODE REVIEW

File: src/main/resources/public/index.html [hunk 1]
```diff
@@ -2,61 +2,144 @@
...
+                        <div class="form-group">
+                            <label for="price">Price</label>
+                            <input type="number" class="form-control" id="price" name="price" required>
+                        </div>
+                        <input type="submit" id="addRoomSubmit" style="display: none;">
+                    </form>
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+                    <button type="button" class="btn btn-primary" onclick="$('#addRoomSubmit').click();">Submit</button>
```
Comment (AvantHof) [inline src/main/resources/public/index.html:139]: Javascript in html?

File: src/main/resources/public/index.html [hunk 2]
```diff
@@ -2,61 +2,144 @@
...
+                                <option value="2">Luxe</option>
+                            </select>
+                        </div>
+                        <div class="form-group">
+                            <label for="size">Size</label>
+                            <input type="number" class="form-control" id="size" name="size" required>
+                        </div>
+                        <div class="form-group">
+                            <label for="price">Price</label>
+                            <input type="number" class="form-control" id="price" name="price" required>
```
Comment (AvantHof) [inline src/main/resources/public/index.html:132]: Defaulten euro tekentje?

**Review by AvantHof — APPROVED** [review]

File: src/main/resources/public/index.html (hunk 1 above)
Comment (jimmy1993k) [reply src/main/resources/public/index.html:139]: Ja

File: src/main/resources/public/index.html (hunk 2 above)
Comment (jimmy1993k) [reply src/main/resources/public/index.html:132]: Goed idee.

**Pull request merged**